index.html

```diff
@@ -127,6 +127,264 @@ <h2 class="job-title title">I'm a software developer</h2>
           </li>
         </ul>
       </div>
+      <div class="down-arrow">
+        <svg
+          width="42"
+          height="42"
+          viewBox="0 0 42 42"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <rect x="1" y="1" width="40" height="40" rx="20" fill="white" />
+          <path
+            fill-rule="evenodd"
+            clip-rule="evenodd"
+            d="M14.2929 17.2929C14.6834 16.9024 15.3166 16.9024 15.7071 17.2929L21 22.5858L26.2929 17.2929C26.6834 16.9024 27.3166 16.9024 27.7071 17.2929C28.0976 17.6834 28.0976 18.3166 27.7071 18.7071L21.7071 24.7071C21.3166 25.0976 20.6834 25.0976 20.2929 24.7071L14.2929 18.7071C13.9024 18.3166 13.9024 17.6834 14.2929 17.2929Z"
+            fill="#172B4D"
+          />
+          <rect x="1" y="1" width="40" height="40" rx="20" stroke="#DFE1E6" />
+        </svg>
+      </div>
+    </section>
+    <section class="work-section">
+      <div class="work-header">
+        <h2 class="work-title">My Recent Works</h2>
+      </div>
+      <div class="bottom-border">
+        <svg
+          width="49"
+          height="4"
+          viewBox="0 0 49 4"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
+        >
+          <rect x="0.5" width="48" height="4" rx="2" fill="#36B37E" />
+        </svg>
+      </div>
+      <div class="projects">
+       <div class="projects-card">
+          <div class="image-holder"></div>
+          <div class="text-container">
+            <div class="project-header">Multi-Post Stories Gain+Glory</div>
```

Comment:
[OPTIONAL] Since this is a project header, you can use an `hx` ie. h2-h6 tag to make it more standard and better SEO optimization instead of the `div` tag. The previous reviewer commented on that, I can see you just changed it from `h1` to `div` that is why I am making it an optional request. Ideally, it could be an `h1` tag, but you should try as much as possible to limit the usage many `h1` tags in a page. I believe this should not be a big issue when you iterate over the card using JavaScript. For now, kindly check and change all this and subsequent ones on lines 168, 183, 198, 213, 228, and 243

Reply:
Thank you very much for pointing it out @jerryowusu, I will keep it in mind